aten/src/ATen/ParallelCommon.cpp

```diff
@@ -18,9 +19,25 @@ namespace at {
 
 namespace {
 
-const char* get_env_var(const char* var_name) {
+const char* get_env_var(
+    const char* var_name, const char* def_value = nullptr) {
   const char* value = std::getenv(var_name);
-  return value ? value : "[not set]";
+  return value ? value : def_value;
+}
+
+size_t get_env_num_threads(const char* var_name, size_t def_value = 0) {
+  try {
+    if (auto* value = std::getenv(var_name)) {
+      int nthreads = std::stoi(value);
+      TORCH_CHECK(nthreads > 0);
+      return nthreads;
+    }
+  } catch (const std::exception& e) {
+    std::ostringstream oss;
+    oss << "Invalid " << var_name << " variable value, " << e.what();
+    TORCH_WARN(oss.str());
```

Comment:
I don't now if there is some aten specific prefix. Also, you don't see a lot of warnings in general as far as I know.

Reply:
we're moving away from AT_ macro to TORCH_ atm